test/integration/configuration/resolveConfig-test.js

```diff
@@ -12,11 +14,13 @@ describe('resolveConfig()', () => {
       },
     },
   });
+  const { config } = resolveConfig(withOverrides({}));
```

Comment:
I don't like how the tests order matters here and how some of the assertions reuse this global config while some of them use a different, local one. It feels like we're not isolating each test properly from each other.

I think each `describe` should have it's own `resolveConfig()` call and dedicated `config` variable, even if that feels like duplication (duplication in tests is often desirable).

Reply:
I've broken test cases for `resolveConfig` more granularly 74091dcc3763bb5a990a71d5017754c2280db1cb